src/librustc/mir/mod.rs

```diff
@@ -2932,3 +2932,5 @@ impl<'tcx> TypeFoldable<'tcx> for Literal<'tcx> {
         }
     }
 }
+
+newtype_index!(LocalWithRegion);
```

Comment:
I would expect this to live in the `borrow_check` somewhere -- probably `nll/liveness_map.rs` (See below)